etl/Snakefile

```diff
@@ -312,7 +314,9 @@ rule raster_temp_file:
         "raster/link/{layer}.tif",
     run:
         layer = get_hazard_layer(wildcards.layer, hazard_layers)
-        os.symlink(os.path.join(config["analysis_data_dir"], "processed_data", layer.path), str(output))
+        original = os.path.abspath(os.path.join(config["analysis_data_dir"], "processed_data", layer.path))
+        print(f"Linking {original} to {output}")
+        os.symlink(original, str(output))
```

Comment:
I'm not sure these changes are needed. The original code wouldn't symlink files on my Mac, until I switched to using absolute paths.

Reply:
Symlinks with relative paths often don't work for me.